Source/WebCore/platform/graphics/gstreamer/mse/AppendPipeline.cpp

```diff
@@ -1389,6 +1418,23 @@ void AppendPipeline::notifyReceivedAllPendingSamples()
         m_sourceBufferPrivate->didReceiveAllPendingSamples();
 }
 
+#if USE(OPENCDM)
```

Comment:
I think all these methods are not needed as they are not used outside the append pipeline.

Reply:
Yes it is used by MediaPlayerPrivateGstreamerMSE flow